### PR TITLE
Added check to the todo item

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
I added a check to the todo item

### CODE REVIEW

File: src/checked.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const checked = () => {
+  const labelList = document.querySelectorAll('.label');
+  const checkBoxes = document.querySelectorAll('.checkbox');
+  for (let i = 0; i < checkBoxes.length; i += 1) {
+    checkBoxes[i].addEventListener('change', () => {
+      // const taskIndex = e.target.getAttribute('data-id');
+      // alert(taskIndex);
```
Comment (carlos-ssh) [inline src/checked.js:7]: - Kindly remove commented code.

File: src/checked.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      if (checkBoxes[i].checked) {
+        console.log(checkBoxes[i], 'mm');
```
Comment (carlos-ssh) [inline src/checked.js:9]: - Kindly Remove console.logs

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @VICTOR-CHUKWU , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wandji20 — APPROVED** [review]
## Hi @VICTOR-CHUKWU  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 

Your project has been approved, however, Keep up the good work.

## Congratulations :+1:  :fireworks: 
## Optional
- [ optional ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .eslintrc.json etc).

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**